src/strategies/unigraph/index.ts

```diff
@@ -0,0 +1,66 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const UNISWAP_SUBGRAPH_URL = {
+  1: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
+};
+
+export async function strategy(network, provider, addresses, options, snapshot) {
+  const params = {
+    users: {
+      __args: {
+        where: {
+          id_in: addresses.map(address => address.toLowerCase())
+        },
+        first: 10,
```

Comment:
Any reason to limit to 10 here? We can have 1000 instead
```suggestion
        first: 1000,
```